build.sbt

```diff
@@ -10,6 +10,7 @@ lazy val `akka-management-root` = project
     `akka-discovery`,
     `akka-discovery-dns`,
     `akka-discovery-kubernetes-api`,
+    `akka-discovery-aws`,
```

Comment:
Should it be called EC2 specifically? Since they also have ECS and other kubernetes specific ones etc

Reply:
your'e right - aws-ec2 sounds like a better choice, since it's more specific ;-)